app/api/actions/github/route.ts

```diff
@@ -324,174 +323,210 @@ export async function POST(request: Request) {
         }
       }
       const count = await addActionCount({ watermelon_user });
-      textToWrite += `### WatermelonAI Summary \n`;
 
-      let businessLogicSummary;
-      if (AISummary) {
-        businessLogicSummary = await getOpenAISummary({
-          commitList,
-          values: {
-            github: github?.data,
-            jira: jira?.data,
-            confluence: confluence?.data,
-            slack: slack?.data,
-            notion: notion?.data,
-            linear: linear?.data,
-            asana: asana?.data,
-          },
-          title,
-          body,
-        });
+      console.log("count: ", count.github_app_uses);
```

Comment:
Please, no logging in prod. 
Use posthog if needed.

Reply:
Thanks